sort/oddeven.go

```diff
@@ -0,0 +1,22 @@
+package sort
+
+func OddEvenSort(data []int){
```

Comment:
Add documentation for OddEvenSort.
Please follow https://github.com/x899/algorithms/pull/2 for reference on how to comment.